test/e2e/minscale_readiness_test.go

```diff
@@ -201,7 +201,7 @@ func privateServiceName(t *testing.T, clients *test.Clients, revisionName string
 func waitForDesiredScale(clients *test.Clients, serviceName string, cond func(int) bool) (latestReady int, err error) {
 	endpoints := clients.KubeClient.Kube.CoreV1().Endpoints(test.ServingNamespace)
 
-	return latestReady, wait.PollImmediate(250*time.Millisecond, time.Minute, func() (bool, error) {
+	return latestReady, wait.PollImmediate(250*time.Millisecond, 3*time.Minute, func() (bool, error) {
```

Comment:
Should we include a comment with your analysis?

Reply:
Well pr kind of links throughout. But sure.